tests/python/pants_test/goal/test_run_tracker.py

```diff
@@ -67,7 +67,7 @@ def test_write_stats_to_json_file(self):
 
     # Execute & verify
     with temporary_file_path() as file_name:
-      self.assertTrue(RunTracker.write_stats_to_json(file_name, stats))
```

Comment:
This was the only place I could find where we used the return value from that method - since we're testing the file exists / matches output below that seemed unnecessary (and I've removed the boolean return values in the method itself).